dask/bytes/s3.py

```diff
@@ -13,15 +14,46 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_s3(storage_options=None):
```

Comment:
Should I copy all arguments from S3FileSystem `__init__` or only the 4 I use in `_get_s3`? Writing it I was thinking about this, but since I have `kwargs` dict on the top level function, I decided to avoid not really necessary argument expansion and just pass a dict.